front/pages/w/[wId]/members/index.tsx

```diff
@@ -608,57 +615,44 @@ function InviteSettingsModal({
       window.location.reload();
     }
   }
-  function validDomain(): boolean {
-    let valid = true;
-    if (domainInput === null) {
-      setAllowedDomainError("");
-    } else {
-      // eslint-disable-next-line no-useless-escape
-      if (!domainInput.match(/^[a-z0-9\.\-]*$/)) {
-        setAllowedDomainError("Allowed domain must be a valid domain name.");
-        valid = false;
-      } else {
-        setAllowedDomainError("");
-      }
-    }
 
-    return valid;
-  }
+  const [, userEmailDomain] = user.email.split("@");
 
   return (
     <Modal
       isOpen={showModal}
       onClose={onClose}
       hasChanged={
-        domainInput !== owner.allowedDomain &&
-        !allowedDomainError &&
-        !domainUpdating
+        isDomainWhitelisted !== Boolean(owner.allowedDomain) && !domainUpdating
       }
-      title="Invitation link settings"
+      title="Workspace settings"
       variant="side-sm"
-      onSave={() => validDomain() && handleUpdateWorkspace()}
+      onSave={() => handleUpdateWorkspace()}
     >
       <div className="mt-6 flex flex-col gap-6 px-2">
-        <div>
-          Any person with a Google Workspace email on corresponding domain name
-          will be allowed to join the workspace.
-        </div>
-        <div className="flex flex-col gap-1.5">
-          <div className="font-bold">Whitelisted email domain</div>
-          <Input
-            className="text-sm"
-            placeholder={"Company domain"}
-            value={domainInput}
-            name={""}
-            error={allowedDomainError}
-            showErrorLabel={true}
-            onChange={(e) => {
-              setDomainInput(e);
-              setAllowedDomainError("");
-            }}
+        <p>
+          Should new users with the whitelisted email domain{" "}
+          <span className="font-bold">
+            {owner.allowedDomain ?? userEmailDomain}
+          </span>{" "}
+          be added to your workspace automatically?
```

Comment:
This is not true yet right? they still need the link?

Reply:
Updated, I used an hardcoded boolean to gate those changes until the sign up flow is being updated top display the old copy.